yabt/buildfile_parser_test.py

```diff
@@ -36,8 +39,9 @@ def test_parser_error(basic_conf, capsys):
     with pytest.raises(SystemExit):
         populate_targets_graph(build_context, basic_conf)
     _, err = capsys.readouterr()
-    ybuild_path = join('tests', 'errors', 'parser-error', 'YBuild')
-    assert '{}", line 6'.format(ybuild_path) in err
+    ybuild_path = re.escape(join('tests', 'errors', 'parser-error', 'YBuild'))
+    logger.info('test_parser_error: {}', err)
```

Comment:
Maybe change this to debug (this is only here so we can debug when the test fails)